src/librustc/mir/mod.rs

```diff
@@ -890,9 +893,7 @@ impl<'tcx> LocalDecl<'tcx> {
         match self.is_user_variable {
             Some(ClearCrossCrate::Set(BindingForm::Var(VarBindingForm {
                 binding_mode: ty::BindingMode::BindByValue(_),
-                opt_ty_info: _,
-                opt_match_place: _,
-                pat_span: _,
+                ..
```

Comment:
Revert; when this pattern is used it is intentional to make things stop compiling on change.